pokemonterminal/wallpaper/adapters/sway.py

```diff
@@ -0,0 +1,14 @@
+from os import environ
+from subprocess import run
+from . import WallpaperProvider as _WProv
+
+
+class SwayProvider(_WProv):
+    def change_wallpaper(path: str):
+        run(["swaymsg", f"output * background {path} fill"], check=True)
```

Comment:
Chances are this won't work with paths that have spaces
Also, Python will emit the command as this: `swaymsg "output * background aaa.jpg fill"`, but I have no idea if swaymsg will handle those quotes correctly.

Reply:
Those quotes are actually needed for swaymsg; below is a gif of me using swaymsg and pokemon-terminal to update my sway wallpaper.



![PokemonTerminal](https://user-images.githubusercontent.com/47319228/68993309-e509a880-0876-11ea-814f-6393de48e306.gif)